src/js/gi/components/heading/PreviewBanner.jsx

```diff
@@ -1,58 +1,38 @@
 import React from 'react';
-import { connect } from 'react-redux';
-import * as actions from '../../actions';
+import moment from 'moment';
 
 class PreviewBanner extends React.Component {
   render() {
-    if (!this.props.show) {
-      return null;
-    }
-
-    const headerDisplacementCSS = `
-      div.header {
-        margin-top: 4em;
-      }
-    `;
-    const tz = { timeZone: 'America/New_York' };
-    const when = new Date(this.props.version.createdAt).toLocaleString('us', tz);
+    const when = moment(this.props.version.createdAt).format('MMM D, YYYY [at] h:mm a [EST]');
 
     return (
       <div className="gi-preview-banner">
-        <style>{headerDisplacementCSS}</style>
-        <div className="outer"/>
-        <div className="inner">
-          <h5>Preview draft</h5>
-          <p>
-            This is what the version of this data from {when} will look like.
-            <span className="actions">
-              <a onClick={this.props.exitPreviewMode}>View live version</a>
-              &nbsp;&nbsp;|&nbsp;&nbsp;
-              <a href={this.props.toolURL}>Go back to the data tool</a>
-            </span>
-          </p>
+        <div className="outer small-12 medium-12 large-9">
+          <div className="inner">
+            <h5>Preview draft</h5>
+            <p>
+              This is what the version of this data from {when} will look like.&nbsp;
+              <span className="actions">
+                <a onClick={this.props.onViewLiveVersion}>View live version</a>
+                &nbsp;&nbsp;|&nbsp;&nbsp;
+                <a href={this.props.toolUrl}>Go back to the data tool</a>
+              </span>
+            </p>
+          </div>
         </div>
       </div>
     );
   }
 }
 
 PreviewBanner.propTypes = {
-  show: React.PropTypes.bool.isRequired,
   version: React.PropTypes.object.isRequired,
-  toolURL: React.PropTypes.string.isRequired,
+  toolUrl: React.PropTypes.string.isRequired,
```

Comment:
This `.isRequired`, but not passed in GiBillApp.jsx (line 67-69). Should it be required if it has a default (line 35)? This may just be convention, I'm not sure.